code/controllers/subsystem/ticker.dm

```diff
@@ -32,6 +32,8 @@ SUBSYSTEM_DEF(ticker)
 	SCRIPTURE_SCRIPT = FALSE, \
 	SCRIPTURE_APPLICATION = FALSE) //list of clockcult scripture states for announcements
 
+	var/list/first_death = list()			//contains info about the first player to die; ckey, name, location, and last words
```

Comment:
Feels like just letting blackbox register these locally too when updating DB would be a better solution.

Reply:
we already have the death table thoug?

Reply:
I mean don't split the two things in two, and store it in the same function. I'd say to skip storing it altogether but single sql query is still less preferable than just storing list of n deaths since that can be used for something else.

Reply:
I'd be up for that, do you know where to start? When I looked at blackbox code I had a difficult time figuring it out.

Reply:
https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/blackbox.dm#L256

Reply:
Add local logging here, then fetch it from wherever you saved it.

Reply:
As a heads-up, if I do it this way, I can't test it to see if it works, as I have no way of checking since the DB isn't enabled locally.